core/templates/services/auth.service.ts

```diff
@@ -81,8 +81,9 @@ class ProdAuthServiceImpl extends AuthServiceImpl {
   }
 
   /** Returns a promise that never resolves or rejects. */
-  async signInWithRedirectAsync(): Promise<void> {
-    return this.angularFireAuth.signInWithRedirect(this.provider);
+  async signInWithPopupAsync(): Promise<void> {
+    this.angularFireAuth.signInWithPopup(this.provider);
```

Comment:
```suggestion
     return this.angularFireAuth.signInWithPopup(this.provider);
```

Reply:
The problem here is `signInWithRedirectAsync` has a return type of void but `signInWithPopup` returns firebase.auth.UserCredential. The previous method `signInWithRedirect` wasn't returning anything.

Reply:
Oh okay. This can be ignored then.